tokens.xml

```diff
@@ -11593,7 +11593,7 @@ Throne of the Dead Three — Reveal the top ten cards of your library. Put a cre
         <card>
             <name>Kaito Shizuki Emblem</name>
             <text>Whenever a creature you control deals combat damage to a player, search your library for a blue or black creature card, put it onto the battlefield, then shuffle.</text>
-            <prop>
+            <prop>https://github.com/SlightlyCircuitous/image-storage/blob/main/karn_emblem_arena.png?raw=true
```

Comment:
what's this?

Reply:
```suggestion
            <prop>
```

Reply:
Not sure. must have accidentally added it pasted it there when I was adding the png link back.